src/main/service/listParseHelperService.ts

```diff
@@ -188,24 +188,28 @@ export class ListParseHelperService {
      * @param session
      */
     public findAndManipulateJudiciary(session: object): string {
-        let judiciaries = '';
-        let foundPresiding = false;
+        const judiciaries = [];
         session['judiciary']?.forEach(judiciary => {
-            if (judiciary?.isPresiding === true) {
-                judiciaries = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
-                foundPresiding = true;
-            } else if (!foundPresiding) {
-                if (ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) !== '') {
-                    judiciaries += ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) + ', ';
-                }
-            }
+            const name = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
+            judiciary?.isPresiding === true ? judiciaries.unshift(name) : judiciaries.push(name);
```

Comment:
You are within the loop for each judiciary, so this judiciary definitely exists, so you can remove the ? operator after the judiciary in this line and the line above. Same with the crime method below.